enterprise/frontend/src/metabase-enterprise/api/metabot.ts

```diff
@@ -1,18 +1,131 @@
 import _ from "underscore";
 
 import type {
+  DeleteSuggestedMetabotPromptRequest,
   MetabotAgentRequest,
   MetabotAgentResponse,
   MetabotApiEntity,
   MetabotEntity,
   MetabotId,
   MetabotInfo,
-  MetabotPromptSuggestions,
   PaginationRequest,
   PaginationResponse,
+  SuggestedMetabotPrompt,
+  SuggestedMetabotPromptsRequest,
+  SuggestedMetabotPromptsResponse,
 } from "metabase-types/api";
 
 import { EnterpriseApi } from "./api";
+import { idTag } from "./tags";
+
+const isTesting = process?.env?.NODE_ENV === "test";
+
+// breaking typing for now, but let's me have the mocked version in browser
+// and unmocked version in jest tests
+const hackRTKQueryDefinition = (definition: any): any => {
```

Comment:
this breaks types but allows me to make real network requests in our unit tests but still use a mocked version of the endpoints when running outside of jest for dev puprposes.

Reply:
This, the fake data, and the `queryFn` definitions will be removed when the endpoints are complete.